src/main/resources/assets/createframed/lang/ru-ru.json

```
@@ -0,0 +1,217 @@
+  "block.createframed.orange_stained_framed_glass_pane": "Оранжевая витражная обрамлённая стеклянная панель",
+  "block.createframed.yellow_stained_framed_glass_pane": "Жёлтая витражная обрамлённая стеклянная панель",
+  "block.createframed.green_stained_framed_glass_pane": "Зелёная витражная обрамлённая стеклянная панель",
+  "block.createframed.lime_stained_framed_glass_pane": "Лаймовая витражная обрамлённая стеклянная панель",
+  "block.createframed.blue_stained_framed_glass_pane": "Синяя витражная обрамлённая стеклянная панель",
+  "block.createframed.light_blue_stained_framed_glass_pane": "Голубая витражная обрамлённая стеклянная панель",
+  "block.createframed.cyan_stained_framed_glass_pane": "Циановая витражная обрамлённая стеклянная панель",
+  "block.createframed.purple_stained_framed_glass_pane": "Фиолетовая витражная обрамлённая стеклянная панель",
+  "block.createframed.magenta_stained_framed_glass_pane": "Пурпурная витражная обрамлённая стеклянная панель",
+  "block.createframed.pink_stained_framed_glass_pane": "Розовая витражная обрамлённая стеклянная панель",
+  "block.createframed.black_stained_framed_glass_pane": "Чёрная витражная обрамлённая стеклянная панель",
+  "block.createframed.gray_stained_framed_glass_pane": "Серая витражная обрамлённая стеклянная панель",
+  "block.createframed.light_gray_stained_framed_glass_pane": "Светло-серая витражная обрамлённая стеклянная панель",
+  "block.createframed.white_stained_framed_glass_pane": "Белая витражная обрамлённая стеклянная панель",
+  "block.createframed.brown_stained_framed_glass_pane": "Коричневая витражная обрамлённая стеклянная панель",
+  "block.createframed.tinted_framed_glass_pane": "Тонированная обрамлённая стеклянная панель",
+
+  "block.createframed.horizontal_red_stained_framed_glass_pane": "Горизонтальная красная витражная обрамлённая стеклянная панель",
+  "block.createframed.horizontal_orange_stained_framed_glass_pane": "Горизонтальная оранжевая витражная обрамлённая стеклянная панель",
+  "block.createframed.horizontal_yellow_stained_framed_glass_pane": "Горизонтальная жёлтая витражная обрамлённая стеклянная панель",
+  "block.createframed.horizontal_green_stained_framed_glass_pane": "Горизонтальная зелёная витражная обрамлённая стеклянная панель",
+  "block.createframed.horizontal_lime_stained_framed_glass_pane": "Горизонтальная лаймовая витражная обрамлённая стеклянная панель",
+  "block.createframed.horizontal_blue_stained_framed_glass_pane": "Горизонтальная синяя витражная обрамлённая стеклянная панель",
+  "block.createframed.horizontal_light_blue_stained_framed_glass_pane": "Горизонтальная голубая витражная обрамлённая стеклянная панель",
+  "block.createframed.horizontal_cyan_stained_framed_glass_pane": "Горизонтальная циановая витражная обрамлённая стеклянная панель",
+  "block.createframed.horizontal_purple_stained_framed_glass_pane": "Горизонтальная Фиолетовая витражная обрамлённая стеклянная панель",
```

There's an unnecessary uppercase here and at line 133
```suggestion
  "block.createframed.horizontal_purple_stained_framed_glass_pane": "Горизонтальная фиолетовая витражная обрамлённая стеклянная панель",
```